mavros_msgs/msg/Tunnel.msg

```diff
@@ -0,0 +1,22 @@
+# Tunnel
+#
+# https://mavlink.io/en/messages/common.html#TUNNEL
+
+uint8 target_system
+uint8 target_component
+uint16 payload_type
+uint8 payload_length
+uint8[128] payload
+
+# MAV_TUNNEL_PAYLOAD_TYPE enum
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_UNKNOWN = 0
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED0 = 200
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED1 = 201
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED2 = 202
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED3 = 203
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED4 = 204
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED5 = 205
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED6 = 206
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED7 = 207
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED8 = 208
+uint16 MAV_TUNNEL_PAYLOAD_TYPE_STORM32_RESERVED9 = 209
```

Comment:
Perhaps `PAYLOAD_TYPE_` prefix is enough. Also i would rather to have it generated by cog from pymavlink.

Reply:
Good point, I'll try to cog it.

Reply:
Do you know how to access the correct enum in pymavlink? I'm trying, based on ADSBVehicle.msg, to do something like
```py
from pymavlink.dialects.v20 import common
# use common.enums['MAV_TUNNEL_PAYLOAD_TYPE']
```
but I can't find any tunnel enums in common.enums:
```py
>>> [x for x in common.enums]
['MAV_COMPONENT', 'MAV_RESULT', 'MAV_ROI', 'UAVCAN_NODE_MODE', 'MAV_PROTOCOL_CAPABILITY', 'MAV_SENSOR_ORIENTATION', 'MOTOR_TEST_ORDER', 'CONDITION_TYPE', 'MAV_STATE', 'LANDING_TARGET_TYPE', 'MAV_MODE_FLAG', 'ADSB_ALTITUDE_TYPE', 'MAV_AUTOPILOT', 'MAV_BATTERY_TYPE', 'MOTOR_TEST_THROTTLE_TYPE', 'MAV_SYS_STATUS_SENSOR', 'ADSB_FLAGS', 'MAV_MISSION_RESULT', 'MAV_CMD_ACK', 'MAV_DO_REPOSITION_FLAGS', 'RTK_BASELINE_COORDINATE_SYSTEM', 'MAV_LANDED_STATE', 'MAV_PARAM_TYPE', 'MAV_COLLISION_THREAT_LEVEL', 'MAV_TYPE', 'MAV_ESTIMATOR_TYPE', 'CAMERA_MODE', 'MAV_COLLISION_ACTION', 'FENCE_ACTION', 'GPS_FIX_TYPE', 'MAV_VTOL_STATE', 'FENCE_BREACH', 'MAV_GOTO', 'MAV_BATTERY_CHARGE_STATE', 'MAV_BATTERY_FUNCTION', 'MAV_CMD', 'MAV_MISSION_TYPE', 'MAV_MODE_FLAG_DECODE_POSITION', 'GPS_INPUT_IGNORE_FLAGS', 'MAVLINK_DATA_STREAM_TYPE', 'MAV_FRAME', 'MAV_MODE', 'UAVCAN_NODE_HEALTH', 'MAV_SEVERITY', 'MAV_ARM_AUTH_DENIED_REASON', 'MAV_POWER_STATUS', 'ESTIMATOR_STATUS_FLAGS', 'SERIAL_CONTROL_FLAG', 'FIRMWARE_VERSION_TYPE', 'MAV_MOUNT_MODE', 'MAV_COLLISION_SRC', 'MAV_DATA_STREAM', 'VTOL_TRANSITION_HEADING', 'MAV_DISTANCE_SENSOR', 'SERIAL_CONTROL_DEV', 'ADSB_EMITTER_TYPE', 'RC_TYPE']
```

Reply:
Maybe your pymavlink is older than the date when that message and enum came in?

Reply:
You're right! I got it now.